ibis/backends/trino/tests/test_client.py

```diff
@@ -185,14 +185,6 @@ def test_list_tables_schema_warning_refactor(con):
 
     assert con.list_tables()
 
-    # Error if user mixes tuple inputs and string inputs for database and schema
-    with pytest.raises(FutureWarning):
-        with pytest.raises(exc.IbisInputError):
-            con.list_tables(database=("tuple", "ohstuff"), schema="str")
-
-    with pytest.warns(FutureWarning):
-        assert con.list_tables(database="tpch", schema="sf1") == tpch_tables
-
```

Comment:
@gforsyth Is this a similar case than the one pointed out above, about missing tests. Do we need to adapt this ones to pass `database="tpch.sf1"` I can't run the trino one locally to try them out

Reply:
Yeah, no issues here because it's no longer possible to "overspecify" a path